hummingbird/ml/_parse.py

```diff
@@ -624,6 +624,8 @@ def _parse_onnx_single_operator(topology, operator):
 
     # Add the operator in the topology.
     alias = get_onnxml_api_operator_name(operator.op_type)
+    if alias is None:
+        alias = "ONNXML" + operator.op_type
```

Comment:
I am curious, in what cases does this happen?

Reply:
StandardScaler is converted into an onnx Scaler when types are float. This operator only supports float, not double. So for double, the scaler is replaced by operator Sub, Mul. Alias is None is this case. The fix I did may be not the best one but it worked in this case.